### ISSUE TITLE
Request : Add setting to disable bigger pause for players

### ISSUE THREAD

**Issue by uniwars** [ironmonk88/monks-little-details#285]
**Is your feature request related to a problem? Please describe.**
Hi, the pause icon is bigger for players and there is no setting to make it like before.

**Describe the solution you'd like**
A setting to turn the feature off, or a slider to choose pause icon size.

Maybe a setting to change the size for the GM too.

**Comment by ironmonk88** [comment]
You mean like this setting?
![image](https://github.com/ironmonk88/monks-little-details/assets/75920956/28b1afff-76ad-43a1-a82a-8ace66a010f4)

**Comment by uniwars** [comment]
Yes but this setting is for moving the icon, not change its size ?
